### DISCUSSION TITLE
разделение на официальные и кастомные режимы

### DISCUSSION THREAD

**Discussion by LooKing070** [Sokol-Games-GmbH/PixelCombats.PublicApi#25] [Announcements]
Я предлагаю создать вкладку выбора режима в серверном листе - "кастомные" или "пользовательские". Куда будут добавляться новые режимы созданные нами, чтобы не путать их с обычным ТДМ и прочими...

**Comment by Kukushechka7** [comment]
Крутая идея

**Comment by kkohno** [comment]
Приняли в работу
